src/phases/execution/ClusteredCollection.yml

```diff
@@ -7,120 +7,153 @@ Description: >
 
 GlobalDefaults:
   RecordIdExtraPaddingParam: &RecordIdExtraPaddingParam {^Parameter: {Name: "RecordIdExtraPadding", Default: 0}}
+  PaddingGenerator: &PaddingGenerator {^FastRandomString: {length: *RecordIdExtraPaddingParam, alphabet: "a"}}
+  Database: &Database test
+  Collection: &Collection Collection0
+  Namespace: &Namespace test.Collection0
+  DocumentCount: &docCount 1000000
+  MaxPhases: &MaxPhases 7
 
-Actors:
 
-# Phase 1: create collection
+Actors:
+# Phase 0: create collection
 - Name: Create
   Type: RunCommand
   Threads: 1
   Phases:
-  - Repeat: 1
-    Database: &db test
-    Operations:
-    - OperationName: RunCommand
-      OperationCommand:
-        create: &coll Collection0
-        clusteredIndex: {key: {_id: 1}, unique: true}
-  - &Nop {Nop: true}
-  - *Nop
-  - *Nop
-  - *Nop
-  - *Nop
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        Operations:
+        - OperationName: RunCommand
+          OperationCommand:
+            create: *Collection
+            clusteredIndex: {key: {_id: 1}, unique: true}
+
+
+# Phase 1: Shard the collection
+- Name: ShardCollection
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        OnlyRunInInstance: sharded
```

Comment:
Would it make sense to configure this on a per-operation basis, rather than on the phase-level? I could see someone wanting to always run one operation, and only sometimes run another.

(If we went with the topology visitor method above, we could construct one visitor with multiple ops on it.)